GameData/RealismOverhaul/Engine_Configs/KVD1_Config.cfg

```diff
@@ -66,7 +66,7 @@
 {
 	%title = KVD-1/CE-7.5
 	%manufacturer = KB Khimavtomatiki (Kosberg) / ISRO
-	%description = Staged combustion hydrolox upper stage engine intended for use on the N-1M and considered for use on Proton and Angara, eventually licensed to India for its GSLV MkI and MkII.	Later versions were developed by India for domestic use on the as the CE-7.5.  The main engine bell is fixed in place, and two verniers are used to provide combined pitch-yaw-roll control; this leads to lower control authority than on stages where the main engine can gimbal.	 This engine runs with a somewhat higher than average O/F ratio, resulting in a denser than average hydrolox stage.
+	%description = A staged combustion hydrolox upper stage engine intended for use on the N-1M and considered for use on Proton and Angara, eventually licensed to India for its GSLV MkI and MkII.	Later versions were developed by India for domestic use on the as the CE-7.5.  The main engine bell is fixed in place and two verniers are used to provide combined pitch-yaw-roll control; this leads to lower control authority than on stages where the main engine can gimbal.	 This engine runs with a somewhat higher than average O/F ratio, resulting in a denser than average hydrolox stage.
```

Comment:
Are the spaces between sentences intended?

Reply:
I believe it was how the original author did newlines. But I think it's better without